### DISCUSSION TITLE
Commas should delimit Cookware

### DISCUSSION THREAD

**Discussion by trentks** [cooklang/spec#91] [General]
Spaces and fullstops delimit, as they should, but so should commas.

It's not uncommon to have a comma after a piece of cookware, and it's a little annoying to have to delimit before it.

By example:
> Crack the @eggs{2} into a #bowl, and then beat with a #whisk.  

Would produce a Cookware that looks like:
- bowl,
- whisk

I don't see any need for a comma to ever be a part of a cookware item's name.

**Comment by dubadub** [comment]
This was fixed in a new rust parser:

```
echo 'Crack the @Eggs{2} into a #bowl, and then beat with a #whisk.' | cook recipe read
  

Ingredients:
  Eggs     2 

Cookware:
  bowl       
  whisk      

Steps:
 1. Crack the Eggs into a bowl, and then beat with a whisk.
     [Eggs: 2]
```
